README.md

```diff
@@ -1037,7 +1037,7 @@ sudo killall -HUP blued
 #### Force Enable Trim
 Enable Trim for non-Apple SSDs. This command is available since Yosemite.
 ```bash
-forcetrim
+trimforce
```

Comment:
i think it's `trimforce enable`:

```bash
$ trimforce help
Usage:
	trimforce enable
	trimforce disable
	trimforce help
```

Reply:
Yes. I assumed the user would learn that once he types the cmd. But feel free to change it into `trimforce enable` as that's what the title says

Reply:
Have you checked if this is true for all earlier versions of macOS since Yosemite? Are you positive the command was always named like this and has never been renamed?

Reply:
It's already been named that in 2015, see https://osxdaily.com/2015/10/29/use-trimforce-trim-ssd-mac-os-x/
Or just google the wrong name. You won't find fitting results.

Reply:
Perfect, thanks for checking.